f2/src/utils/formatAnalystEmail.js

```diff
@@ -64,10 +64,20 @@ const formatVictimDetails = (data) => {
 
   let postalCity = ''
   let postalProv = ''
-
-  if (data.location.postalCode) {
-    postalCity = zipcodes.lookup(data.location.postalCode).city
-    postalProv = zipcodes.lookup(data.location.postalCode).state
+  try {
+    let location = zipcodes.lookup(data.location.postalCode)
+    if (location === undefined) {
+      postalCity = 'Location lookup is not found'
+      postalProv = 'Location lookup is not found'
+    } else {
+      postalCity = location.city
+      postalProv = location.state
+    }
+  } catch (error) {
+    //logging
+    console.error(error)
+    postalCity = 'Location lookup failed'
+    postalProv = 'Location lookup failed'
```

Comment:
although var error=2  in the example above is declared in the local scope and it should remain local, in our case we are re-assigning it to globally scoped var and i'm not sure if 'Location lookup failed' will ever be assigned

Reply:
Confirmed, it works!! it's always good to test twice  :)